policyengine_us/variables/gov/local/co/denver/dhs/property_tax_relief/eligibility/co_de_property_tax_relief_homeowner_eligible.py

```diff
@@ -0,0 +1,27 @@
+from policyengine_us.model_api import *
+
+
+class co_de_property_tax_relief_homeowner_eligible(Variable):
+    value_type = float
+    entity = SPMUnit
+    unit = USD
+    label = "Eligible for the homeowner Denver Property Tax Relief"
+    definition_period = YEAR
+    reference = "https://denvergov.org/files/content/public/v/37/government/agencies-departments-offices/agencies-departments-offices-directory/denver-human-services/be-supported/additional-assistance/property-tax-relief/denver-property-tax-relief-program-year-2021-rules.pdf"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.local.co.denver.dhs.property_tax_relief
+        has_property_taxes = add(spm_unit, period, ["real_estate_taxes"]) > 0
+        has_elderly_or_disabled = spm_unit("has_usda_elderly_disabled", period)
+        count_dependents = add(spm_unit, period, ["tax_unit_dependents"])
+        has_dependent = count_dependents > 0
+        elderly_or_disabled_homeowners = (
+            has_elderly_or_disabled | has_dependent
+        ) & has_property_taxes
+
+        income = spm_unit("spm_unit_net_income", period)
```

Comment:
It's based on income after taxes and transfers? Check the income definition.

Reply:
It's based on income before taxes. It's the total of earnings/wages, social security income, and other income.

Reply:
spm_unit_net_income is after taxes and transfers

Reply:
@hua7450 we can use `spm_unit_market_income` then

Reply:
as in all programs, we define a list of countable income sources based on the statutory language or program documents

Reply:
I add a income list based on program instruction, the legal code does not defined income for this program.